marshmallow/utils.py

```diff
@@ -343,3 +343,15 @@ def callable_or_raise(obj):
 def get_func_args(func):
     """Return a tuple of argument names for a function."""
     return inspect.getargspec(func).args
+
+
+def get_callable_args(func):
```

Comment:
Is there any reason we should keep both `get_func_args` and `get_callable_args`?